src/components/screens/consul/Debates/DebateDetail.js

```diff
@@ -53,14 +53,14 @@ export const DebateDetail = ({ data, onRefresh, route, navigation }) => {
   );
 
   useEffect(() => {
-    getConsulUser().then((val) => {
-      if (val) return setUserId(JSON.parse(val).id);
+    getConsulUser().then((userInfo) => {
+      if (userInfo) return setUserId(JSON.parse(userInfo).id);
     });
   }, []);
 
   const { control, handleSubmit, reset } = useForm({
     defaultValues: {
-      comment: ''
+      comment: null
```

Comment:
same here as asked before, why changed to `null`?

Reply:
i thought null was more accurate both in terms of readability and in terms of disk space.